### DISCUSSION TITLE
Spotube windows client not working

### DISCUSSION THREAD

**Discussion by ItsAdi22** [KRTirtho/spotube#1829] [General]
I downloaded the latest windows client of spotube. Whenever I try to launch it, nothing happens (there's no UI) . In the background it tries to access the location permissions.

I have also tried downloading the nightly version of spotube but it also does the same. Am I the only one who is facing this issue or some of you are facing it too?

I'm running windows 11 pro edition. version 22H2 (just for context).

**Comment by Hi-Phile** [comment]
Try using version 3.6.0, there's been many people encountering issues with it not working (either not starting, or nothing displayed) when trying to use 3.8.0+33 with Windows 11.

Other possible issues to check:
https://github.com/KRTirtho/spotube/issues/1853
https://github.com/KRTirtho/spotube/issues/1786

**Comment by ItsAdi22** [comment]
Hey there, 
Version 3.6.0 seems to work for me 
Thank You :)
